airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/context.py

```diff
@@ -60,6 +61,7 @@ def __init__(
         s3_build_cache_access_key_id: Optional[str] = None,
         s3_build_cache_secret_key: Optional[str] = None,
         concurrent_cat: Optional[bool] = False,
+        run_step_options: RunStepOptions = RunStepOptions(),
```

Comment:
I think it would make sense to expose this parameter at the `PipelineContext` level for reusability. Wdyt?

Reply:
We do! Im just passing it through to the pipeline context here